source/content/guides/terminus-drupal-site-management.md

```diff
@@ -12,9 +12,10 @@ tags: [terminus, drush]
 type: guide
 permalink: docs/guides/:basename
 contributors: [erikmathy]
-
+reviewed: "2022-12-13"
```

Comment:
@tom, there are references to d7 and d9.  see preview: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/terminus-drupal-site-management